### PR TITLE
Update Estimating Geological Metrics.md

### PR DESCRIPTION
**Pull request by sarikasu**

### CODE REVIEW

**Comment by sarikasu** [comment]
@markwhiting, I made small formatting changes to this over the summer, can you squash and merge this since it already had a second review?

**Review by markwhiting — APPROVED** [review]

**Comment by markwhiting** [comment]
Approved (but its cleaner for history if you press the merge button?)

**Pull request merged**
